src/ship.cpp

```diff
@@ -1,25 +1,27 @@
 #include "ship.h"
 
+#define PI 3.14159265f
```

Comment:
just add 
#include <math.h>
instead of defining your own constants for Pi etc

Reply:
bummmmpppp

Reply:
Fine.